CONTRIBUTING.md

```
@@ -0,0 +1,120 @@
+
+2. **Make your changes.** Follow the conventions below.
+
+3. **Run checks before committing:**
+   ```bash
+   npm run lint
+   npm run test
+   npm run build
+   ```
+
+4. **Commit** using [conventional commits](https://www.conventionalcommits.org/):
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)